src/OrchardCore.Modules/OrchardCore.ReverseProxy/Views/ReverseProxySettings.Edit.cshtml

```diff
@@ -0,0 +1,34 @@
+@model ReverseProxySettingsViewModel
+
+<fieldset class="form-group" asp-validation-class-for="EnableXForwardedFor">
```

Comment:
Maybe add a link to the reference documentation https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/proxy-load-balancer?view=aspnetcore-2.1